pkg/kubelet/cri/streaming/portforward/httpstream.go

```diff
@@ -170,7 +170,7 @@ func (h *httpStreamHandler) removeStreamPair(requestID string) {
 func (h *httpStreamHandler) requestID(stream httpstream.Stream) string {
 	requestID := stream.Headers().Get(api.PortForwardRequestIDHeader)
 	if len(requestID) == 0 {
-		klog.V(5).Infof("(conn=%p) stream received without %s header", h.conn, api.PortForwardRequestIDHeader)
+		klog.V(5).InfoS("Connection stream received without requestID header", "connection", h.conn, "portForwardRequestIDHeader", api.PortForwardRequestIDHeader)
```

Comment:
I think that PortForwardRequestIDHeader is a const, correct me if I'm wrong. If so then we should either:

Depend on name of header in message. The "without **requestID header**" part of message already points to which header it is.
```suggestion
		klog.V(5).InfoS("Connection stream received without requestID header", "connection", h.conn)
```
or pass it as simple `header` key

```suggestion
		klog.V(5).InfoS("Connection stream received without header", "connection", h.conn, "header", api.PortForwardRequestIDHeader)
```

Reply:
Maybe the simpler the better